src/transform/ancestor.js

```diff
@@ -118,7 +118,7 @@ function canBeLifted(doc, range) {
   }
 }
 
-// :: (Node, Pos, ?Pos) → bool
+// :: (Node, Pos, ?Pos) → ?{found: any, range: any}
```

Comment:
Currently, I'm working around ~10 actual bugs and obvious short-comings in Flow. Aside from that, you are regularly using patterns that are very difficult to prove statically, for example seemingly-infinite loops, methods that always stop execution (`Error.raise`), `if` clauses that assert non-nullness in a non-obvious way, etc. On a more fundamental level, you regularly amend classes which doesn't work to well with static analysis.